flake.nix

```
@@ -5,6 +5,7 @@
     nixosModules = {
       dell-aspire-4810t = import ./acer/aspire/4810t;
       asus-fx504gd = import ./asus/fx504gd;
+      asus-ga401 = import ./asus/ga401;
```

I wonder if it should have the series in its attribute?

```suggestion
      asus-zephyrus-ga401 = import ./asus/zephyrus/ga401;
```

---

ga401 is uniquely identifiable and fc504gd didn't have one so I didn't add one